code/modules/mob/mob.dm

```diff
@@ -3,6 +3,9 @@
 	GLOB.dead_mob_list -= src
 	GLOB.alive_mob_list -= src
 	focus = null
+	if(length(progressbars))
+		stack_trace("[src] destroyed with elements in its progressbars list")
```

Comment:
Maybe add a comment here saying it is being deleted due to the signal `COMSIG_PARENT_QDELETING` being listened to by the progress bars